frontend/src/lib/gql/gql-client.ts

```diff
@@ -97,11 +95,22 @@ class GqlClient {
       context: {fetch, ...context}
     });
 
-    return derived(resultStore, (result) => {
-      this.throwAnyUnexpectedErrors(result);
-      return result;
-    });
+    if (browser) {
+      return derived(resultStore, (result) => {
+        this.throwAnyUnexpectedErrors(result);
+        return result;
+      });
+    } else {
+      /**
+       * We kill vite if we validate each query result and throw in the urql pipeline, but we shouldn't ever need to, because:
```

Comment:
```suggestion
       * We kill node if we validate each query result and throw in the urql pipeline, but we shouldn't ever need to, because:
```
vite doesn't run in prod, only node, wanted to clarify this doesn't just effect dev